pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -357,12 +362,9 @@ def addons(self):
 
         for addon_uid in self._addon_uids:
             addon_key = "addon-" + addon_uid
-            if self._data[addon_key]["repository"] != '.':
-                repository = self._data[addon_key]["id"]
-                packages = "{}/{}".format(repository, PACKAGES_DIRECTORY)
-            else:
-                repository = self._data[addon_key]["repository"]
-                packages = PACKAGES_DIRECTORY
```

Comment:
I don't believe there is a case when addon can point to a main repo, or is there?

Reply:
Good question. I have no idea. I guess we can wait until we find one that does.

Reply:
I don't know, I did it because I thought it could be a possibility, but it was just a guess